ui/src/services/TaskService.ts

```diff
@@ -0,0 +1,98 @@
+import { collection, getDocs, limit, query, QuerySnapshot, where } from 'firebase/firestore';
+import { db } from '@lib/firebase';
+import { Task } from 'types/Task';
+import { getAuth } from 'firebase/auth';
+import { fsDataConverter } from 'consts/fsDataConverter';
+import { Score } from 'types/Score';
+import { TaskScore } from 'types/TaskScore';
+
+class TaskService {
```

Comment:
ciekawe podejście z klasą, ale chyba niepraktyczne. musimy za każdym razem tworzyć instancje tej klasy, ale nie korzystamy z żadnych jej dodatkowych pól. Wygląda mi to na dobrego kandydata na refactor do folderu api gdzie będziemy mieli te wszystkie funkcje wyeksportowane pojedynczo zamiast z klasy